lib/modules/datasource/github-runners/index.ts

```diff
@@ -9,17 +9,26 @@ export class GithubRunnersDatasource extends Datasource {
    * Only add stable runners to the datasource. See datasource readme for details.
    */
   private static readonly releases: Record<string, Release[] | undefined> = {
-    ubuntu: [{ version: '22.04' }, { version: '20.04' }, { version: '18.04' }],
+    ubuntu: [
+      { version: '22.04' },
+      { version: '20.04' },
+      { version: '18.04', isDeprecated: true },
+      { version: '16.04', isDeprecated: true },
+    ],
     macos: [
       { version: '13' },
       { version: '13-large' },
       { version: '13-xlarge' },
       { version: '12' },
       { version: '12-large' },
```

Comment:
The upstream readme mentions `macos-latest`, `macos-12`, `macos-latest-large`, and `macos-12-large`. We don't have all those labels in our list yet.

https://github.com/actions/runner-images/blob/main/README.md#available-images

Reply:
The `latest` are non-op for us as we have no pinning for these right now.    

Regarding tags `macos-` will be prefixed to the versions in the manager, if I recall correctly.

Reply:
Oh okay, just wanted to mention the apparently missing labels. 😄